doc/modules/tree.rst

```diff
@@ -535,7 +535,9 @@ where :math:`X_m` is the training data in node :math:`m`
     * https://en.wikipedia.org/wiki/Decision_tree_learning
 
     * https://en.wikipedia.org/wiki/Predictive_analytics
-
+    
+    * `Understanding the decision tree structure`<https://scikit-learn.org/dev/auto_examples/tree/plot_unveil_tree_structure.html/>
```

Comment:
It might be better suited for the `Tips on practical use` section

Also the link should be ``` `Understanding... <http...>`_ ``` (maybe we can reference an example with sphinx-gallery? I'm not sure)

Reply:
I agree with your suggestion on adding the link to the `Tips on practical use`, thanks for pointing out the misplaced quote. By 

> we can reference an example with sphinx-gallery

Do you mean to say that we can add the output of the referred code or the code blocks

Reply:
> It might be better suited for the `Tips on practical use` section

I agree and I would also add a link to the example in the `Example` subsection of the `Classification` section.

Reply:
>Do you mean to say that we can add the output of the referred code or the code blocks

I was thinking about using a sphinx ref rather than a hard html link, like ```:ref:`sphx_glr_auto_examples_tree_plot_iris_dtc.py` ```

Reply:
I have removed it from the Reference and added it to the `Tips on Practical use` section in the latest commit